apic_ml2/neutron/services/l3_router/l3_apic.py

```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
```

Comment:
ok, does the user get a reasonable error message when that happens? Reporting a DB constraint violation is not very user friendly.